src/map/clif.c

```diff
@@ -453,13 +454,18 @@ bool clif_send(const void* buf, int len, struct block_list* bl, enum send_target
 
 		case AREA:
 		case AREA_WOSC:
+		case AREA_DEAD:
 			if (sd && bl->prev == NULL) //Otherwise source misses the packet.[Skotlex]
 				clif->send (buf, len, bl, SELF);
 			/* Fall through */
 		case AREA_WOC:
 		case AREA_WOS:
+			if (type == AREA_DEAD)
```

Comment:
probably better add AREA_DEAD into battle.conf like other areas?

Reply:
I thought this is just a 1 line edit ?
just clif->send ... AREA_SIZE *2

what's the point of having this as a setting...

Reply:
Initially I thought about making it a config as you suggested, but I think it'll be a bit confusing and cause issues, because if people make AREA_DEAD smaller than AREA_SIZE, or not big enough, the issue will be back.
Letting it there without a config and using a value that will be safe for pretty much any area_size will make people think better before changing it (if someone ever wants to change that).

Reply:
@AnnieRuru point what option can be changed
for some one 2x area is too big for some one else too small.
Any hard coded value is bad.

@guilherme-gm if you afraid what some one can set wrong value, you can add comment about what value here better to set. Or even not add any value to config, and left it default value by default, and custom if some one add line into config.

Reply:
I agree with @4144 let people set the AREA_DEAD themselves and document it in a way that people will choose reasonable numbers, like the recommended one 2 * AREA_SIZE. (Which it can be by default anyways)

This would also make this function more aesthetically pleasing than it would be with this PR's change so far.